packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol

```diff
@@ -51,11 +51,11 @@ contract L2ToL1MessagePasser is Semver {
 
     /**
      * @notice Emitted any time a withdrawal is initiated. An extension to
-     *         WithdrawalInitiated so that the interface is maintained.
+     *         MessagePassed to allow for a 4th indexed argument.
      *
      * @param hash The hash of the withdrawal
      */
-    event WithdrawalInitiatedExtension1(bytes32 indexed hash);
+    event MessagePassedExtension1(bytes32 indexed hash);
```

Comment:
Comment to stop mergify

Reply:
I think we can consolidate these events, the workflows are going to involve more getting the event by user who is withdrawing and then pulling the hash from the event

Reply:
I'm open to this via a followup PR

Reply:
yeah, agree that's better